contracts/governance/utils/IVotes.sol

```diff
@@ -1,13 +1,28 @@
 // SPDX-License-Identifier: MIT
 // OpenZeppelin Contracts (last updated v4.5.0) (governance/utils/IVotes.sol)
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.18;
 
 /**
  * @dev Common interface for {ERC20Votes}, {ERC721Votes}, and other {Votes}-enabled contracts.
  *
  * _Available since v4.5._
  */
 interface IVotes {
+    /**
+     * @dev Lookup to future votes is not available.
+     */
+    error VotesFutureLookup(uint256 timepoint, uint48 clock);
+
+    /**
+     * @dev The signature used has expired.
+     */
+    error VotesExpiredSignature(uint256 expiry);
+
+    /**
+     * @dev The signature's nonce differs from the account's current nonce.
+     */
+    error VotesInvalidNonce(address account, uint256 currentNonce);
```

Comment:
InvalidNonce is going to be something shared between different contract. IMO it should be a "shared" error in the "nounces" module.